metrics/metrics.go

```diff
@@ -135,7 +136,13 @@ func appendDefaults(prefix string) (string, error) {
 }
 
 // stubbed out for testing
-var hostname = os.Hostname
+var hostname = func() (string, error) {
+	out, err := exec.Command("hostname", "-f").Output()
```

Comment:
No, it sucks. If my host doesn't have fqdn then akubra doesn't start (and reports much saying message:time="Mar 10 13:33:06.209397" level=fatal msg="exit status 1").
I prefer it to fallback to os.Hostname if fqdn is not available.